modules/monitoring-common-terms.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-overview.adoc
+
+:_content-type: REFERENCE
+[id="openshift-monitoring-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+alerts::
+
+Alertmanager::
+The Alertmanager service handles alerts received from Prometheus. Alertmanager is also responsible for sending the alerts to external notification systems.
+
+Alerting rules::
+Alerting rules contain a set of conditions that outline a particular state within a cluster. Alerts are triggered when those conditions are true. An alerting rule can be assigned a severity that defines how the alerts are routed.
+
+Cluster Monitoring Operator::
+The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus instances, the Thanos Querier, the Telemeter Client, and metrics targets and ensures that they are up to date. The CMO is deployed by the Cluster Version Operator (CVO).
+
+Cluster Version Operator::
+
+CoreDNS::
+
+config map::
+A config map provides a way to inject configuration data into pods. You can reference the data stored in a config map in a volume of type `ConfigMap`. Applications running in a pod can use this data.
+
+
+
+Container::
+It is a lightweight and executable image that comprises software and all its dependencies. Containers virtualize the operating system, as a result, you can run containers anywhere from a data center to a public or private cloud to even a developer’s laptop.
+
+
+custom resource (CR)::
+A CR is extension of the Kubernetes API. You can create custom resources.
+
+Elasticsearch::
+
+etcd::
+
+Federation endpoint::
+
+Fluentd::
+
+HAProxy::
+
+Image registry::
+
+Kubelets::
+Runs on nodes and reads the container manifests. Ensures that the defined containers have started and are running.
+
+Kubernetes API server::
+
+Kubernetes controller manager::
+Kubernetes controller manager governs the state of the cluster.
+
+Kubernetes scheduler::
+Kubernetes scheduler allocates pods to nodes.
+
+labels::
+You can use labels, which are key-value pairs, to organise and select subsets of objects, such as a pod.
+
+Metering::
+
+metrics::
+
+metric target::
+
+monitoring dashboards::
+
+monitoring stack::
```

Comment:
no explanation for
`metrics`, `metric target:`, `monitoring dashboards`, `monitoring stack`